src/input/UnicodeKeymap.cc

```diff
@@ -88,6 +88,9 @@ UnicodeKeymap::UnicodeKeymap(string_view keyboardType)
 		auto buf = file.mmap();
 		parseUnicodeKeyMapFile(
 			string_view(reinterpret_cast<const char*>(buf.data()), buf.size()));
+		if (!msxChars.has_value()) {
+			msxChars.emplace("MSXVID.TXT");
+		}
```

Comment:
Fine for now. But in the (near?) future we'll require that all unicodemap files must specify the msx-encoding.
(Already done on the WIP-keyboard-stuff branch).